include/drivers/dma.h

```diff
@@ -31,7 +31,8 @@ extern "C" {
 enum dma_channel_direction {
 	MEMORY_TO_MEMORY = 0x0,
 	MEMORY_TO_PERIPHERAL,
-	PERIPHERAL_TO_MEMORY
+	PERIPHERAL_TO_MEMORY,
+	PERIPHERAL_TO_PERIPHERAL /*only supported in NXP EDMA*/
```

Comment:
please move all the changes in this file to a separate commit

Reply:
ok